src/rprblender/properties/world.py

```diff
@@ -238,6 +238,11 @@ class RPR_EnvironmentProperties(RPR_Properties):
         description="Background override type",
         default='SPHERE',
     )
+    backplate_crop: bpy.props.BoolProperty(
```

Comment:
another semantic point, but maybe should change the description of this property. This is actually a resize, not a crop, no?

Reply:
It is more related to cropping edges of backplate image for keeping aspect ratio of the image in render. That's why I think crop better corresponds here instead resize.